jackson-modules/jackson-conversions-3/src/test/java/com/baeldung/jackson/absentvsnull/JacksonAbsentVsNullUnitTest.java

```diff
@@ -0,0 +1,130 @@
+package com.baeldung.jackson.absentvsnull;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Test;
+
+import com.baeldung.jackson.absentvsnull.model.Sample;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.exc.MismatchedInputException;
+
+class JacksonAbsentVsNullUnitTest {
+
+    private static final String ID_FIELD = "id";
+    private static final String PRIMITIVE_FIELD = "amount";
+    private static final ObjectMapper DEFAULT_MAPPER = new ObjectMapper();
+
+    private static final Sample ZERO_ARG_SAMPLE = new Sample();
```

Comment:
I think it's a bit misleading to call these constants since their values can change.

I'd recommend you give your learners less to read by doing:

```java
Sample empty = new Sample();
```

Even better, doing `new Sample()` is so simple, I'd strongly recommend simply doing that in each test instead of sharing an instance. This makes it easier to run tests concurrently and it reduces the amount of scrolling a reader needs to do to comprehend a given test.